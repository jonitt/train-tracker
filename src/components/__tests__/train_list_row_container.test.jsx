import { shallow } from "enzyme";
import React from "react";
import TrainListRowContainer from "../train_list_row/train_list_row_container.jsx";

/*
  component @props:
    isHeader: is row a header line
    cancelled: is train cancelled
    delayed: is train delayed
    delayedTime: new time train is delayed to
    name: trains name / code number
    departureStation: station leaving from at first
    arrivalStation: station arriving to at last
    time: time train is scheduled for
*/
describe("TrainListRowContainer", () => {
  it("renders correctly", () => {
    const wrapper = shallow(
      <TrainListRowContainer
        delayed={true}
        delayedTime="12:12"
        name="train"
        departureStation="helsinki"
        arrivalStation="Oulu"
        time="12:10"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
