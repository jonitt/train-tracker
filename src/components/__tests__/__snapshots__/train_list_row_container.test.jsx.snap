// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrainListRowContainer renders correctly 1`] = `ShallowWrapper {}`;
